Just typos (#51)
w3f/polkadot-wiki: docs/polkadot/learn/staking.md

# Staking Information

> Polkadot will use NPoS(Nominated Proof-of-Stake) consensus, which will have a limited amount of slots available at the begining for someone who is interested in running a validator to maintain the network. NPoS is designed to optimize nominator/validator for maximum chain security. Basically, DOTs holder gets pay more for nominating multiple validators to ensure the network will not be heavily centralized on few validators. As a validator, it only takes the role of either validating blocks or guaranteeing the finality of the chain to earn rewards, while nominator will choose a set of potential validators candidate by indicating their DOTs as weight. 

Reviewer: ```suggestion
Polkadot will use NPoS (Nominated Proof-of-Stake) consensus, which will have a limited amount of slots available at the beginning for someone who is interested in running a validator to maintain the network. NPoS is designed to optimize nominator/validator for maximum chain security. Basically, DOTs holder get paid more for nominating multiple validators to ensure the network will not be heavily centralized on only a few validators. As a validator, it only takes the role of either validating blocks or guaranteeing the finality of the chain to earn rewards, while nominator will choose a set of potential validators candidate by indicating their DOTs as weight. 
```

> Any validators can indicate their intention to be a validator candidate in anytime. It will be added to a waiting queue for nominators to elect and become active in the next era if a validator get enough nomination support.      

Reviewer: ```suggestion
Any potential validators can indicate their intention to be a validator candidate. Validators who signal intention will be added to a waiting queue for nominators to elect and become active in the next era if a validator get enough nomination support.      
```

> ## Accounts

Reviewer: ```suggestion

## Accounts
```

Add whitespace

> There are two different accounts for managing your funds, which is 

Reviewer: ```suggestion
There are two different accounts for managing your funds, these are
```

> Stash & Controller account. 

Reviewer: ```suggestion
Stash & Controller accounts. 
```

> **Stash:** It is reponsible for holding an owner's funds used for staking that are controlled by a cold wallet and bonding DOTs token to the controller. All bonded DOTs are locked. It has to be waited a certain of time in order to access the locked funds.

Reviewer: ```suggestion
**Stash:** The stash is responsible for holding an owner's funds used for staking (could be held in a cold wallet) and bonding DOTs to the controller. All bonded DOTs are locked. After unbonding, users must wait a certain amount of time in order to access the locked funds.
```

**Controller:** It is actively reponsible for managing the account operations such as expressing an interest to switch between roles (validator, nominator, idle).

## Operations

_TODO_ - Step-By-Step Guide for nominating potential validators 

## Slashing

> Slashing will be applied if a validator has been reported to offline a number of times. Once slashing is determined, a value will be deducted from the balance of the validator and all the nominators voted for this validator.

Reviewer: ```suggestion
Slashing will be applied if a validator has been reported to be offline for a number of times. Once slashing is determined, a value will be deducted from the balance of the validator and all the nominators that have voted for this validator.
```

## Reward Distribution

Based on the the current configuration in PoC-3 Alexander testnet, it takes 1 hour to finish an era that means rewards will be distributed to the validators and nominators per hour. 

!!! info
    PER_ERA * BLOCK_TIME = **Reward Distribution Time**

    600 * 6 = 3600 = 1 Hour

    ***These parameters can be changed by proposing a referendum***

> Validator can create a reward cut that is not shared with the nominators. After value gets deducted, the remaining portion is based on the proportional to their staked value to split in between validaor and all of the nominators who had a vote for this validator.

Reviewer: ```suggestion
Validator can create a cut of the reward that is not shared with the nominators. After value gets deducted, the remaining portion is based on their staked value and split between the validator and all of the nominators who have voted for this validator.
```

For example, assume reward is 100 DOTs.

> If a validator specify `validator_payment = 50 DOTs`, the remaining 50 would split in between the validator and their nominators based on their share they had.

Reviewer: ```suggestion
A validator may specify `validator_payment = 50 DOTs` and the remaining 50 DOTs would split between the validator and their nominators based on the portion of stakes they had.
```

> Rewards can be used the same account(controller) to keep accumulating the rewards or stash account (inceaseing the staked value/not increasing the staked value). Also, it is possible to top-up/withdraw partial bonded DOTs without having to complete un-stake everything.

Reviewer: ```suggestion
Rewards can be used the same account (controller) to keep accumulating the rewards or stash account (increasing the staked value/not increasing the staked value). Also, it is possible to top-up/withdraw partial bonded DOTs without having to complete un-stake everything.
```
